java/src/main/java/ai/rapids/cudf/Rmm.java

```diff
@@ -136,6 +136,55 @@ public static boolean isInitialized() throws RmmException {
    */
   public static native long getTotalBytesAllocated();
 
+  /**
+   * Returns the maximum amount of RMM memory (Bytes) outstanding during the
+   * lifetime of the process.
+   *
+   * Note that this result is meaningful when a single thread is using the GPU, or
+   * when we have joined all threads and CUDA synchronized with all streams.
```

Comment:
I would remove this comment, as I think it's more confusing than helpful.  It's also more specific to the Spark plugin's use of RMM and streams and not the concern of this interface.  The "lifetime of the process" part of the above comment already covers what's important for this API.